packages/block-library/src/reset.scss

```diff
@@ -7,7 +7,7 @@
 
 // We use :where to keep specificity minimal.
 // https://css-tricks.com/almanac/selectors/w/where/
-:where(.editor-styles-wrapper) {
+html :where(.editor-styles-wrapper) {
```

Comment:
Given this is a reset stylesheet, I'm inclined towards the `0-0-1` specificity as well. 

Perhaps we could further isolate the change here by extracting only the rules needed to override the `common.css` `body` styles under the restored selector. For example:

```css
html :where(.editor-styles-wrapper) {
	/**
	* The following styles revert to the browser defaults overriding the WPAdmin styles.
	* This is only needed while the block editor is not being loaded in an iframe.
	*/
	font-family: serif; // unfortunately initial doesn't work for font-family.
	font-size: initial;
	line-height: initial;
	color: initial;
	// Many themes with white backgrounds load editor styles but fail to also provide
	// an explicit white background color, assuming a white editing canvas.
	// So to match browser defaults, we provide a white default here as well.
	background: #fff;
}
```

The rest could remain under `:where(.editor-styles-wrapper)`.

There is also [another PR](https://github.com/WordPress/gutenberg/pull/56649) around the editor styles wrapper that might be worth taking a look at to see how these changes might be affected.

Reply:
If you want to isolate the one element, you could use the `html &` method I mentioned in https://github.com/WordPress/gutenberg/issues/57176#issuecomment-1861193823

However, a PR like that is more complex than I should try to create within the browser, so someone else should make it.

Plus, the comment about not being in an iframe is inaccurate, so that sentence could be removed too.